### PR TITLE
feat: Refactoring the SD-Factory

### PR DESCRIPTION
**Pull request by adkumar1**
- Refactoring the SD-Factory to support number of formats (Catena-22-10, GaiaX, FC) as output. 
-  It shall be fully compatible on level of Catena-22-10 with previous versions. 
- Creates a payload and send it unsigned to the CH

### CODE REVIEW

File: DEPENDENCIES
Comment (almadigabor) [inline DEPENDENCIES:1]: Please open IP issue tickets for restricted dependencies.

**Review by almadigabor — CHANGES_REQUESTED** [review]

File: DEPENDENCIES
Comment (dvasunin) [reply DEPENDENCIES:1]: yes, we have some libraries marked as "restricted" due to the version upgrade. Can you please raise the tickets for us as we do not have "committer" priviledges.

**Comment by almadigabor** [comment]
I was running the dash tool and got the following output. Please track the generated IP issue tickets found below in the links:

```
[INFO] This content is either not correctly mapped by the system, or requires review.
[INFO] A review is required for maven/mavencentral/com.danubetech/key-formats-java/1.6.0.
[INFO] A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/10950 .
[INFO] A review is required for maven/mavencentral/info.weboftrust/ld-signatures-java/1.2.0.
[INFO] A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/10951 .
[INFO] A review is required for maven/mavencentral/com.goterl/lazysodium-java/5.1.1.
[INFO] A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/10952 .
[INFO] A review is required for maven/mavencentral/com.danubetech/verifiable-credentials-java/1.1.0.
[INFO] A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/10953 .
[INFO] A review is required for maven/mavencentral/decentralized-identity/jsonld-common-java/1.1.0.
[INFO] A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/10954 .
```

File: DEPENDENCIES
Comment (almadigabor) [reply DEPENDENCIES:1]: sure, tickets were raised, see them in the comments

**Comment by almadigabor** [comment]
I'll merge this commit but release must not be created from this as the `DEPENDENCIES` file contains restricted libraries (with IP issue numbers).

**Pull request merged**